sousmot/sousmotapp/templates/registration/signup.html

```diff
@@ -0,0 +1,73 @@
+{% extends "sousmotapp/base.html" %}
+
+{% block title %}Sign Up{% endblock %}
+
+{% block content %}
+
+    <section class="hero has-text-centered">
+        <div class="hero-body">
+            <h1 class="title">Sign Up</h1>
+            <p class="subtitle">Nice to meet you, join-us !</p>
+
+            <div class="box">
+                <form method="post">
+                    {% csrf_token %}
+                    <!-- Username -->
+                    <div class="field">
+                        <label for="id_username" class="label">Username</label>
+                        <div class="control">
+                            <input id="id_username" type="text" name="username" class="input" required>
+                        </div>
+                    </div>
+                    <!-- Password 1 -->
+                    <div class="field">
+                        <label for="id_password1" class="label">Password</label>
+                        <div class="control">
+                            <input id="id_password1" type="password" name="password1" class="input" required>
```

Comment:
```suggestion
                            <input id="id_password1" type="password" name="password1" class="input" minlength="8" required>
```
Maybe use HTML attributes so the user know before sending that their password doesn't match ?